app/presenters/chat_channel_detail_presenter.rb

```diff
@@ -39,4 +40,9 @@ def membership_users(memberships)
       }
     end
   end
+
+  def inviation_link(chat_channel)
```

Comment:
It also looks like everywhere we are missing a "t" in "invitation", we should probably update that.